File: archive/first-letter-to-appear-twice.js
const repeatedCharacter = function(s) {
  let mem = {}
  let ans = ''
  for (let i = 0; i < s.length; i++) {
    if (mem[s[i]] === undefined) {
      mem[s[i]] = true
    } else {
      ans = s[i]
      break
    }
  }

  return ans
};

File: go-archive/first-letter-to-appear-twice.go
func repeatedCharacter(s string) byte {
    mem := map[byte]bool{}

    for i:=0; i < len(s); i++ {
        if _, isHas := mem[s[i]]; isHas {
            return s[i]
        } else {mem[s[i]] = true}
    }

    return 'a'
}